main/actions.js

```diff
@@ -1,10 +1,13 @@
 const { Ooniprobe } = require('./utils/ooni/ooniprobe')
+const { getAutorunHomeDir } = require('./utils/paths')
+const fs = require('fs-extra')
 const log = require('electron-log')
 const Sentry = require('@sentry/electron')
 
 const hardReset = () => {
   const ooni = new Ooniprobe()
   log.info('hardReset: performing a hard reset of the installation')
+  fs.removeSync(getAutorunHomeDir())
```

Comment:
I think that what we ought to be doing is calling `ooniprobe reset --force` with the `OONI_HOME` environment variable set to `getAutorunHomeDir`.

That way, if there is any additional cleanup operation that needs to happen on hard reset, we can implement it inside of probe-cli directly and not have to remember to keep it in sync between the two directories.